### PR TITLE
Fix: Fire Base uses wrong model when sold while badly damaged

### PR DESCRIPTION
**Pull request by commy2**
https://github.com/TheSuperHackers/GeneralsGamePatch/assets/6576312/276aa982-b3a3-4bf3-b7ba-939c34ab34de

### CODE REVIEW

File: Patch104pZH/Design/Changes/v1.0/2199_fire_base_badly_damage_sold_model.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+date: 2023-08-03
+
+title: Fixes Fire Base model when badly damaged and sold
```
Comment (xezon) [inline Patch104pZH/Design/Changes/v1.0/2199_fire_base_badly_damage_sold_model.yaml]: Fixes incorrect USA Fire Base model when badly damaged and sold

File: Patch104pZH/Design/Changes/v1.0/2199_fire_base_badly_damage_sold_model.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+changes:
+  - fix: Issue where Fire Base damage state is not properly displayed when being sold while badly damaged.
```
Comment (xezon) [inline Patch104pZH/Design/Changes/v1.0/2199_fire_base_badly_damage_sold_model.yaml]: The badly damaged USA Fire Base now properly shows when being sold.

**Review by xezon — APPROVED** [review]
very good

File: Patch104pZH/Design/Changes/v1.0/2199_fire_base_badly_damage_sold_model.yaml (hunk 1 above)
Comment (commy2) [reply Patch104pZH/Design/Changes/v1.0/2199_fire_base_badly_damage_sold_model.yaml]:
```suggestion
title: Fixes incorrect USA Fire Base model when badly damaged and sold
```

File: Patch104pZH/Design/Changes/v1.0/2199_fire_base_badly_damage_sold_model.yaml (hunk 2 above)
Comment (commy2) [reply Patch104pZH/Design/Changes/v1.0/2199_fire_base_badly_damage_sold_model.yaml]:
```suggestion
  - fix: The badly damaged USA Fire Base now properly shows when being sold.
```

**Pull request merged**